docs/configuration-discovery-instance-filters.md

```diff
@@ -0,0 +1,101 @@
+# Configuration: instance filters
+
+Sometimes it is desirable to exclude particular hosts from discovery.
+It may be the case for large, complex environments where auxiliary nodes like gh-ost, Debezium, Tungsten, etc., are present.
+
+A node can be excluded by specifying its host's name regex pattern. When the Orchestrator detects such a node and its host name matches one of the given patterns, the node is skipped during the discovery process:
+
+```json
+{
+  "DiscoveryIgnoreHostnameFilters": [
+    "a_host_i_want_to_ignore[.]example[.]com:5000",
+    ".*[.]example2.com:5000",
+    "192.168.0.1:6000"
+  ]
+}
+```
+
+Regexp filters to apply to prevent auto-discovering new replicas. Usage: unreachable servers due to firewalls, applications which trigger binlog dumps:
```

Comment:
You can also use the regex patterns to prevent auto-discovering new replicas. These replicas may be on unreachable servers due to firewall rules.